docs/02-database.ipynb

```diff
@@ -7,10 +7,10 @@
    "source": [
```

Comment:
<p>We should say "... within ~2 degrees of your provided location in the last 60 days"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-workshop/pull/16/discussion/#comment-2470568515'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>